src/cls/IPM/Main.cls

```diff
@@ -960,12 +961,16 @@ ClassMethod GetListModules(pNamespace As %String = {$Namespace}, pSearch As %Str
         Set tParams = tParams _ $ListBuild("?")
         Set tArgs($Increment(tArgs)) = tItem
       }
-      Set tQuery = tQuery _ " WHERE name IN (" _ $ListToString(tParams, ", ") _ ")"
+      Set tQuery = tQuery _ " AND name IN (" _ $ListToString(tParams, ", ") _ ")"
```

Comment:
Note: you don't need to fix this, but all this code would be much cleaner if it was just name %INLIST ? where ? is the $ListBuild list from line 955.